assets/style.css

```diff
@@ -0,0 +1,1017 @@
+/* General definitions */
+
+@font-face {
+  font-family: "Roboto Condensed";
+  src: url(fonts/RobotoCondensed-Regular.ttf);
+}
+
+html * {
+  font-family: "Roboto Condensed", arial, sans-serif;
+}
+
+body {
+  background-color: #f4f5f6;
+  min-width: 450px;
+}
+
+.container,
+.container-fluid {
+  max-width: 1310px;
+}
+
+.bg-primary {
+  background-color: #f72210 !important;
+}
+
+.text-primary {
+  color: #f72210 !important;
+}
+
+.border *,
+.border {
+  border: solid 2px black !important;
+}
+
+.border-none {
+  border: none !important;
+}
+
+/* links */
+
+a:hover,
+a:link,
+a:active,
+a:visited {
+  color: currentcolor;
+  text-decoration: none;
+}
+
+.link-c-red a:hover,
+.link-c-red:hover {
+  color: #f72210;
+}
+
+.link-c-red2 a:hover,
+.link-c-red2:hover {
+  color: #f92859;
+}
+
+.link-c-black a:hover,
+.link-c-black:hover {
+  color: black !important;
+}
+
+.link-u-red a:hover,
+.link-u-red:hover {
+  text-decoration: underline red !important;
+}
+
+.link-b-gray a:hover,
+.link-b-gray:hover {
+  background-color: #dbdcdd;
+}
+
+/* main */
+
+main {
+  max-width: 1310px;
+  margin-left: auto;
+  margin-right: auto;
+  margin-top: 0;
+}
```

Comment:
This is a great opportunity to rely on bootstrap's [spacing](https://getbootstrap.com/docs/4.5/utilities/spacing/) utililties.

Reply:
Fixed ; )

Reply:
Although I did my best, there remain some issues: 

1.- There are no defined bootsrap classes for `w-md-100`,  `w-lg-100`, for example. 
2.-  The definition for spacing utilities are defined in `rem` units but most of the times I need to use `px` when comparing to the original layout. 
3.- Some are definition for seudo-classes such as   `:after` or  `focus`. 
4.- In some cases I think it's better to target all elements at once instead of assigning a bootstrap class to each one of the elements such as for example:  `more-stories-items img { flex-shrink: 0}`  instead of writing  `flex-shrink: 0` to each  `img` element.